tests/ha/ha_cts_support_server.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "basetest";
+use strict;
+use testapi;
+use lockapi;
+use mmapi;
+
+sub run() {
+    my $domainname  = get_var("CLUSTERNAME") . ".ha-test.qa.suse.de";
+    my $host1       = "host1";
+    my $host2       = "host2";
+    my $cts_timeout = get_var("CTS_TIMEOUT");
+    wait_for_children_to_start;
```

Comment:
Actually these lines are not really needed, only to check, that support server DNS works, and it always works, because it uses qcow2 image :)